modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+ Error counts only when the flow finishes with an error propagation.
+ By default all the errors will count as a failure. |  |
+| Errors Threshold a| Number | How many errors, of those considered by this circuit, have to occur for the circuit to be opened. |  | x
+| Trip Timeout a| Number | How long will the circuit remain open once the errorsThreshold is reached |  | x
+| Trip Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS | The time unit bound to the tripTimeout | MILLISECONDS |
+|===
+
+[[repeatable-in-memory-stream]]
+=== Repeatable In Memory Stream
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size a| Number | The amount of memory that will be allocated to consume the stream and provide random access to it. If the stream contains more data than can be fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  |
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  |
+| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  |
+| Buffer Unit a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB | The unit in which all these attributes are expressed |  |
+|===
+
+[[repeatable-file-store-stream]]
+=== Repeatable File Store Stream
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed then it will start to buffer the content on disk. |  |
+| Buffer Unit a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB | The unit in which maxInMemorySize is expressed |  |
+|===
+
+[[RedeliveryPolicy]]
+=== Redelivery Policy
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Redelivery Count a| Number | The maximum number of times a message can be redelivered and processed unsuccessfully before triggering process-failed-message |  |
+| Use Secure Hash a| Boolean | Whether to use a secure hash algorithm to identify a redelivered message. |  |
+| Message Digest Algorithm a| String | The secure hashing algorithm to use. If not set, the default is SHA-256. |  |
+| Id Expression a| String | Defines one or more expressions to use to determine when a message has been redelivered. This property may only be set if useSecureHash is false. |  |
```

Comment:
GLOBALLY lood for "specifies" and "defines" to see if the descriptions are parallel: verb-based or noun-based. Try to make peer units of text parallel.

Set only if useSecureHash is false.